frontend/pages/ManageControlsPage/SetupExperience/cards/RunScript/RunScript.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from "react";
+import { useQuery } from "react-query";
+
+import { DEFAULT_USE_QUERY_OPTIONS } from "utilities/constants";
+import mdmAPI from "services/entities/mdm";
+
+import SectionHeader from "components/SectionHeader";
+import DataError from "components/DataError";
+import Spinner from "components/Spinner";
+
+import CustomLink from "components/CustomLink";
+
+import RunScriptPreview from "./components/RunScriptPreview";
+import RunScriptUploader from "./components/RunScriptUploader";
+import RunScriptCard from "./components/RunScriptCard";
+import DeleteRunScriptModal from "./components/DeleteRunScriptModal";
+
+const baseClass = "run-script";
+
+interface IRunScriptProps {
+  currentTeamId: number;
+}
+
+const RunScript = ({ currentTeamId }: IRunScriptProps) => {
```

Comment:
Can we rename this (and other sub-components) in this PR from `RunScript` to `SetupExperienceScript`?

Reply:
yep that makes sense. will make that change